composer.lock

```diff
@@ -360,49 +304,47 @@
                 "serialize",
                 "tokenizer"
             ],
-            "time": "2015-03-11 20:06:43"
+            "time": "2015-12-05 17:17:57"
         },
         {
             "name": "laravel/framework",
-            "version": "v5.1.24",
+            "version": "v5.3.9",
```

Comment:
The composer should reflect the lowest Laravel version. If I would merge these changes, all the people using this package with a lower version of Laravel would not be able to use it anymore. Since there is no reason to use `5.3`, this will not be merged.